mne_connectivity/viz/_3d.py

```diff
@@ -20,7 +20,13 @@
 
 @fill_doc
 def plot_sensors_connectivity(
-    info, con, picks=None, cbar_label="Connectivity", n_con=20, cmap="RdBu"
+    info,
+    con,
+    picks=None,
+    cbar_label="Connectivity",
```

Comment:
We try to add `*` nowadays, maybe this is a good spot?
```suggestion
    *,
    cbar_label="Connectivity",
```